iron-input.html

```diff
@@ -103,6 +103,25 @@
       'keypress': '_onKeypress'
     },
 
+    registered: function() {
+      if (/Firefox|Trident/.test(navigator.userAgent)) {
```

Comment:
How would you detect this?  Like, create a disabled input at startup, fire an event, and try/catch it?  That's probably fine.  http://jsbin.com/sakasu/edit?html,console,output

Alternatively, would it just be simpler to patch `dispatchEvent` unconditionally?  Not sure there's any real perf concern since input events generally aren't high-frequency...?